project_2/README.md

```diff
@@ -0,0 +1,38 @@
+## Проект 2: Алгоритм контроля перегрузки на основе кредитов на C++ (ExpressPass)
```

Comment:
Not needed for issues https://github.com/Cloud-Storage-Team/algnet/issues/30 and https://github.com/Cloud-Storage-Team/algnet/issues/31.